### PR TITLE
feat: allow to run a subset of pr_tester

### PR DESCRIPTION
**Pull request by zdtsw**
- currently run default 8 11 17 18 19 takes a lot resources
	- allow to use "run tests quick" to only run build on 17
	- this still generates all jobs as a basic check

test https://ci.adoptopenjdk.net/job/build-scripts-pr-tester/job/openjdk-build-pr-tester/1719/console

DO NOT MERGE TILL RELEASE(code freeze) DONE

### CODE REVIEW

**Comment by zdtsw** [comment]
run tests quick

**Review by andrew-m-leonard — APPROVED** [review]
lgtm

**Comment by andrew-m-leonard** [comment]
@zdtsw wondering if we could add the version to run on maybe? eg.
```
run tests quick 18
```

**Comment by zdtsw** [comment]
run tests quick 11

**Comment by zdtsw** [comment]
run tests quick 9,19

**Comment by zdtsw** [comment]
```GitHub pull request #365 of commit 8491ead90366bff610aeda3216d17de6104624d0, no merge conflicts.
Setting status of 8491ead90366bff610aeda3216d17de6104624d0 to PENDING with url https://ci.adoptopenjdk.net/job/build-scripts-pr-tester/job/openjdk-build-pr-tester/1720/ and message: 'Building & Testing PR...'
Using context: pipeline-build-check
Obtained pipelines/build/prTester/kick_off_tester.groovy from git https://github.com/andrew-m-leonard/ci-jenkins-pipelines.git
[Pipeline] Start of Pipeline
[Pipeline] node
Running on [C3jenkins](https://ci.adoptopenjdk.net/computer/C3jenkins/) in /home/jenkins/.jenkins/workspace/build-scripts-pr-tester/openjdk-build-pr-tester
not sure why it tries to git clone ci-jenkins-pipeline.git from your repo? @andrew-m-leonard

**Comment by zdtsw** [comment]
```
GitHub pull request #365 of commit 8491ead90366bff610aeda3216d17de6104624d0, no merge conflicts.
Setting status of 8491ead90366bff610aeda3216d17de6104624d0 to PENDING with url https://ci.adoptopenjdk.net/job/build-scripts-pr-tester/job/openjdk-build-pr-tester/1720/ and message: 'Building & Testing PR...'
Using context: pipeline-build-check
Obtained pipelines/build/prTester/kick_off_tester.groovy from git https://github.com/andrew-m-leonard/ci-jenkins-pipelines.git
[Pipeline] Start of Pipeline
[Pipeline] node
Running on [C3jenkins](https://ci.adoptopenjdk.net/computer/C3jenkins/) in /home/jenkins/.jenkins/workspace/build-scripts-pr-tester/openjdk-build-pr-tester

```
not sure why it tried to git clone from your repo andrew-m-leonard/ci-jenkins-pipelines.git ? @andrew-m-leonard

**Comment by zdtsw** [comment]
run tests quick 11

**Comment by zdtsw** [comment]
run tests quick

**Comment by zdtsw** [comment]
run tests

**Comment by zdtsw** [comment]
run tests quick

**Comment by zdtsw** [comment]
run tests

**Comment by zdtsw** [comment]
run tests quick 8,11,17,18,19

**Comment by zdtsw** [comment]
@andrew-m-leonard could you another look at this PR?

**Review by andrew-m-leonard — APPROVED** [review]

File: pipelines/build/prTester/pr_test_pipeline.groovy [hunk 1]
```diff
@@ -124,9 +124,29 @@ class PullRequestTestPipeline implements Serializable {
...
+                run tests quick 8           run jdk8
+                run tests quick 11,17,19    run jdk11, 17 and 19
+            */
+            String[] commentsList=context.params.ghprbCommentBody.trim().split('run tests quick')
+            switch (commentsList.size()) {
+                case 0:
+                    javaVersions = [17]
+                    break
+                case 1:
+                    javaVersions= javaVersions
```
Comment (karianna) [inline pipelines/build/prTester/pr_test_pipeline.groovy:142]:
```suggestion
                    javaVersions = javaVersions
```

**Review by karianna — APPROVED** [review]
LGTM

**Pull request merged**